src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -13,6 +13,7 @@ public class Shooter extends SubsystemBase {
   private final TalonFX flywheelmotor1 = new TalonFX(CAN.flywheel1);
   private final TalonFX flywheelmotor2 = new TalonFX(CAN.flywheel2);
   private double m_rpm;
+  private double m_headingOffset;
```

Comment:
Where is this used?

Reply:
used to offset 2nd flywheel (only if we have 2 side flywheels)